backend/tests/v5/unit/middleware/dataConverter/inputs/teamspaces/projects/models/commons/revisions.test.js

```diff
@@ -113,7 +114,7 @@ Quota.sufficientQuota.mockImplementation((ts) => (ts === 'noQuota' ? Promise.rej
 RevisionsModel.isTagUnique.mockImplementation(() => true);
 
 const testValidateNewRevisionData = () => {
-	const standardBody = { tag: '123', description: 'this is a model', importAnim: false };
+	const standardBody = { tag: '123', description: 'this is a model', importAnimations: false, timezone: 'Europe/Berlin' };
```

Comment:
could we add tests where we pass in `timezone` as:
1. null
2. some other type (like a number)

Reply:
if I pass null, it interprets it as undefined for some reason and then in the validation its the same as not including timezone at all :/ can I skip null ?

Reply:
no, it would be good to add it, it's more for the situations where people alter the code in the future, if it break this test then we know